src/main/java/org/ibp/api/java/impl/middleware/study/StudyEntryMapper.java

```diff
@@ -78,4 +83,20 @@ protected void configure() {
 			}
 		});
 	}
+
+	public static List<StudyEntryDto> map(final List<Germplasm> sourceList, final Map<Integer, String> gidDesignationMap,
+		Integer entryNo, final List<Integer> germplasmDescriptorIds, final Integer entryTypeId, final Map<Integer, String> gidCrossMap) {
```

Comment:
Is entryNo referring to the starting number to use? If that's the case, I think we need to rename it and make it final @mderamos